### PR TITLE
docs: erpc

### PR DESCRIPTION
**Pull request by kasrakhosravi**
**What changed? Why?**

Adding [eRPC](https://github.com/erpc/erpc) to Base docs.

Screenshot 2024-11-07 at 10 29 14

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by wbnns — CHANGES_REQUESTED** [review]
@kasrakhosravi 

Thanks for this PR. Apologies, we'd need multiple options before we can create an additional page. Would eRPC fit within the current Node Providers page?

File: apps/base-docs/docs/tools/node-providers.md [hunk 1]
```diff
@@ -139,6 +139,19 @@ hide_table_of_contents: true
 
 ---
 
+## eRPC
+
+[eRPC](https://github.com/erpc/erpc) is a fault-tolerant EVM RPC proxy and re-org aware permanent caching solution. It is built with read-heavy use-cases in mind such as data indexing and high-load frontend usage.
+
+To get started, visit the [develoepr documentation](https://docs.erpc.cloud/).
```
Comment (wbnns) [inline apps/base-docs/docs/tools/node-providers.md:146]: s/develoepr/developer/

**Comment by wbnns** [comment]
@kasrakhosravi 

Could you please fix the typo in the latest update?

**Pull request closed without merging**